src/clang/misc.jl

```diff
@@ -0,0 +1,6 @@
+typealias FILE Void # not sure if this works
+
+# workaround error in common.jl
+# https://github.com/OSGeo/gdal/blob/d7451d90134f6ca4dea8551b3a5d8591d8e4efad/gdal/port/cpl_port.h#L512-L534
+# this is wrapped as: const CPL_LSBPTR16 = x
+x = nothing
```

Comment:
Hehe I wanted to keep the generated files without manual edits, so I used this. Though it's not very clean. Perhaps we can automatically catch these in `wrap_gdal.jl` later.